ENGAGEHF/HeartHealth/HeartHealth.swift

```diff
@@ -31,6 +32,9 @@ struct HeartHealth: View {
                     }
                 }
                 .background(Color(.systemGroupedBackground))
+                .sheet(item: $addingMeasurement, onDismiss: { addingMeasurement = nil }) { measurementType in
+                    AddMeasurementView(for: measurementType)
+                }
```

Comment:
You could format this in multiple closures:
```suggestion
                .sheet(
                    item: $addingMeasurement, 
                    onDismiss: {
                         addingMeasurement = nil
                     },
                    content: { measurementType in
                        AddMeasurementView(for: measurementType)
                    }
                )
```